src/components/table/draggable-table-cell/draggable-table-cell.scss

```diff
@@ -0,0 +1,16 @@
+.draggable-table-cell {
+  width: 21px;
```

Comment:
It's quite a small value for something fixed, is it describing a touch area (should be bigger if so)